python/redmonster/sandbox/verify.py

```diff
@@ -885,14 +885,14 @@ def cmass_failure_vs_sn_all(self,sn_max=10,nbins=29):
         rbins = n.zeros(nbins)
         ibins = n.zeros(nbins)
         zbins = n.zeros(nbins)
-        for i in xrange(nbins):
+        for i in range(nbins):
             rbins[i] = (rbinedges[i+1]+rbinedges[i])/2.
             ibins[i] = (ibinedges[i+1]+ibinedges[i])/2.
             zbins[i] = (zbinedges[i+1]+zbinedges[i])/2.
-        rhist = rhist / map(float,rtotal)
-        ihist = ihist / map(float,itotal)
-        zhist = zhist / map(float,ztotal)
-        for i in xrange(nbins):
+        rhist = rhist / list(map(float,rtotal))
+        ihist = ihist / list(map(float,itotal))
+        zhist = zhist / list(map(float,ztotal))
```

Comment:
`rtotal`, `itotal`, and `ztotal` are numpy arrays from an earlier `np.histogram()` call.  `list(map(float,rtotal))` would be cleaner as just `rtotal.astype(float)`